packages/react/src/global-nav/GlobalNav.tsx

```diff
@@ -0,0 +1,57 @@
+import * as RadixCollapsible from "@radix-ui/react-collapsible";
+import { forwardRef, useState } from "react";
+import { type ReactNode } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { Flex } from "../flex";
+import { GlobalNavContext } from "../global-nav-context";
+import { GlobalNavItem } from "../global-nav-item";
+import { IconAngleLeft } from "../icons/IconAngleLeft";
+import { extractSprinkles } from "../sprinkles";
+import * as styles from "./GlobalNav.css";
+type GlobalNavProps = BoxProps<
```

Comment:
Let's add a Blank line before it.
```suggestion

type GlobalNavProps = BoxProps<
```